robotoff/workers/tasks/import_image.py

```diff
@@ -917,6 +926,12 @@ def generate_ingredient_prediction_data(
         # available
         if entity["lang"]:
             lang_id = entity["lang"]["lang"]
+            # Skip if the language code is not a valid 2-letter code
+            if not is_valid_language_code(lang_id):
+                logger.warning(
+                    f"Skipping ingredient parsing for invalid language code: {lang_id}"
```

Comment:
Can you add a unit test for `generate_ingredient_prediction_data` to test this behaviour?